addon/globalPlugins/winWizard.py

```diff
@@ -415,6 +417,8 @@ def splitToStacks(self):
 							str(self[windowNumber])
 						)
 					)
+				log.info(f'Windows hidden in the same stack: {hiddenInCurrentStack}')
+				log.info(f'Yielded stack number: {Stack(stackNumber, sorted(hiddenInCurrentStack, key=lambda slot: slot.presentationalNumber))}')
 				yield Stack(stackNumber, sorted(hiddenInCurrentStack, key=lambda slot: slot.presentationalNumber))
```

Comment:
Same here i.e. assign into a variable, use it when logging and then finally yield.